pkg/storaged/format-dialog.jsx

```diff
@@ -306,11 +342,18 @@
                           ]
                       })
         ],
+        update: function (dlg, vals, trigger) {
+            if (trigger == "at_boot")
+                dlg.set_options("at_boot", { explanation: mount_explanation[vals.at_boot] });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-075406-1d0286c4-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)